src/components/RoomDialog.tsx

```diff
@@ -78,7 +78,7 @@ function RoomModal({
           </div>
           <div className="RoomDialog-usernameContainer">
             <label className="RoomDialog-usernameLabel" htmlFor="username">
-              Username:
+              {t("labels.name")}
```

Comment:
Does it make sense to have an ESLint rule banning JSXText nodes so this mistake doesn’t happen again?

Reply:
Is there a rule like that?! I don't want to add plugins for it..

Reply:
https://eslint.org/docs/rules/no-restricted-syntax

Reply:
Let's add it.

Reply:
https://github.com/excalidraw/excalidraw/pull/1367